src/staveline.ts

```diff
@@ -261,15 +261,14 @@ export class StaveLine extends Element {
   }
 
   // Renders the `StaveLine` on the context
-  draw(): this {
+  draw(): void {
```

Comment:
There is also a `draw(): this` in `stavesection.ts`.

Should that one also be changed to void? In the repo, draw() almost always returns void.

Reply:
done